src/plots/cartesian/layout_attributes.js

```diff
@@ -666,6 +666,40 @@ module.exports = {
         editType: 'ticks',
         description: 'Sets the width (in px) of the zero line.'
     },
+
+    showdividers: {
```

Comment:
Adding axis-wide `showdividers`, `dividercolor` and `dividerwidth`. Would this be enough for the first iteration, or per-category-level setting required?